### PR TITLE
Init/project setup

### PR DESCRIPTION
**Pull request by diegoro1**
Added the necessary files for development and updated the README.md with a little bit of info and our GitHub links. If this is the desired format then make sure to add/send me your GitHub link 🙂

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
+# The Team 7 Project (Name TBA)
 
+A website that connects the homeless to shelter.
+
+## Running Locally
+
+Redirect to ./App and run:
+
+```bash
+mvn package exec:java
```
Comment (perrycate) [inline README.md:10]: I can't think of any reason why it wouldn't, but did you double-check that this actually works?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,22 @@
...
+```
+The application should then run on port 8080.
+
+
+## Team 7
+[Perry Cate](https://github.com/perrycate)
```
Comment (perrycate) [inline README.md]: Bah put me at the bottom and add some parenthesis like "advisor" or something. Y'all are the ones doing the real work ;)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,22 @@
...
+
+[Cameron Payton](https://github.com)
```
Comment (perrycate) [inline README.md]: His github is https://github.com/cpayton-source

File: README.md [hunk 4]
```diff
@@ -1,2 +1,22 @@
...
+
+[Diego Rodrigues](https://github.com/diegoro1)
+
+[Jacinda Soto](https://github.com)
```
Comment (perrycate) [inline README.md]: Her github is https://github.com/jacinda22

**Review by perrycate — APPROVED** [review]
Looks good to me once you add those tweaks to the readme!

**Review by cpayton-source — APPROVED** [review]
Hey Diego! This looks great!

File: README.md (hunk 1 above)
Comment (diegoro1) [reply README.md:10]: Just tripled checked and the project opens on port 8080 👍

**Pull request merged**